lib/knapsack_pro/test_case_detectors/rspec_test_example_detector.rb

```diff
@@ -18,7 +18,18 @@ def generate_json_report
         ensure_report_dir_exists
         remove_old_json_report
 
-        test_file_paths = KnapsackPro::TestFileFinder.call(test_file_pattern)
+        test_file_paths =
```

Comment:
TODO: This should be replaced with a request to API to get the slow test files.

We should save list of slow test files to the disk in a separate file along with json report with test examples for those slow test files.

If there is no test files returned from API or the request failed then assume empty list of slow test files.

Reply:
We should add new method to `lib/knapsack_pro/test_case_detectors/rspec_test_example_detector.rb` to read slow test file paths from file on disk.